services/comedian/hashtag_similarity.py

```diff
@@ -1,18 +1,72 @@
-import json
+import json, sys, os
+from scipy.special import gdtr
+sys.path.append(os.path.join(os.path.dirname(__file__), "../util"))
+from loopy import Loopy
+
+
 
 class HashtagClusters:
-    def __init__(self, min_posts):
+    #prior_timeperiod_ms is the time, in miliseconds, to search for previous hashtag clusters of the same type.
+    def __init__(self, min_posts, query_url, start_time_ms, prior_ms=604800000, likelihood_threshold=0.7):
         self.hash_groups = {}
         self.min_post = min_posts
+        self.total_posts = 0
+        self.url = query_url if query_url[-1] == "/" else query_url+"/"
+        self.prior_ms = prior_ms
+        self.start_ms = int(start_time_ms)
+        self.l_thresh = likelihood_threshold
+
+    def get_priors(self, term):
+        q_start_time = self.start_ms - self.prior_ms
+        query_params = [{
```

Comment:
we have an index on ```end_time_ms``` currently, so we prob use that field for queries in another service. not sure where. anyone know? using ```start_time_ms``` in this change. possible inconsistency?